spec/javascripts/test_spec.js

```diff
@@ -57,6 +57,27 @@ describe("Basic testing", function() {
     blurredLocation.setBlurred(false);
     expect(blurredLocation.isBlurred()).toBe(false);
   });
+
+  it("Checks if input listeners change maps position to the entered latitude and longitude", function() {
+
+    var lat = document.getElementById("lat");
+    var lng = document.getElementById("lng");
+
+    lat.value = 20;
```

Comment:
I did all the things in jquery but still the tests won't pass, the value is not changing it seems. The previous typeError is resolved although. But still I didn't get why it worked, could you tell me please